✏️ Style: re-format source codes except member

Turbstructor opened the pull request:
# 개요

`Member` 도메인을 제외하고 모든 코드들에 대해 리포매팅/리팩토링 진행

# 작업사항

- [x] Member 제외한 도메인들에 대해 리포매팅 진행
- [x] (필요할 경우) 리팩토링 진행
  - [x] `DTO`의 `request`와 `response` 패키지 분리
- [x] 오류 해결
  - [x] 게시글 수정/삭제 시 올바르지 않은 자원으로도 게시글을 접근할 수 있는 오류 수정
  - [x] 채널 삭제 시 내부에 있는 모든 게시판/글/댓글을 삭제하는 기능 구현

# 관련 이슈

- close #27

Turbstructor committed:
✏️ Style: re-format source codes except member

This commit is extraordinally large.
- This commit fixes styles of all source codes (except those within
  `domain.member`) to comply to code conventions.
- This commit implements deleting channel which involves removing all
  sub-boards, sub-posts and sub-comments.
- This commit refactors some source code. For example, all `request`s
  and `response`s are separated to separate packages `dto.request` and
  `dto.response`.

hellou8363 commented on src/main/kotlin/spartacodingclub/nbcamp/kotlinspring/project/team4ighting/spring4gamer/domain/comment/service/CommentService.kt:
```diff
@@ -82,20 +84,21 @@ class CommentService(
         )
 
-        return comment.toResponse()
+        return commentRepository.save(targetComment).toResponse()
```
save를 추가 호출하는 이유가 있을까요?

Turbstructor replied:
더티 체킹을 하지만 (수정한 걸 '저장'한다는 의미로) *명시적으로* `save`를 불렀습니다!

Turbstructor committed:
🦺 Fix: add omitted @Valid annotations

Some functions in `PostController` and `ChannelAdminController` didn't have `@Valid` annotation in `request`s. This commit fixes this issue.

KangBaekho10 commented:
확인했습니다. 클린 코드 작성에 도움이 됐습니다. 감사합니다.

taeaeaeae reviewed (approved):
리팩토링 진행하느라 고생하셨어요 통일된 느낌이 보기 좋아요

hellou8363 commented on src/main/kotlin/spartacodingclub/nbcamp/kotlinspring/project/team4ighting/spring4gamer/domain/post/service/PostService.kt:
```diff
@@ -87,9 +86,10 @@ class PostService(
         viewCountUp(post, request, response)
 
-        return post.toResponse()
+        return postRepository.save(post).toResponse()
```
이 부분도 왜 save가 필요한지 궁금합니다.

hellou8363 reviewed (approved):
확인했습니다.

KangBaekho10 reviewed (approved):
확인했습니다. 클린 코드 작성에 도움이 됐습니다. 감사합니다.
